client/src/components/grants/Card.tsx

```diff
@@ -9,7 +9,7 @@ import { projectPath } from "../../routes";
 import { getProjectImage, ImgTypes } from "../../utils/components";
 import { getProjectURIComponents } from "../../utils/utils";
 import { getNetworkIcon, networkPrettyName } from "../../utils/wallet";
-import TextLoading from "../base/TextLoading";
```

Comment:
if TextLoading is not used anymore anywhere else, can we remove the file?

Reply:
Yeah, that's removed already